src/lib/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from './convertPercentageColor';
```

Comment:
We could just import `convertPercentageColor` in extractBrush and avoid `export *` here.

Reply:
[1fbdbb1](https://github.com/software-mansion/react-native-svg/pull/2363/commits/1fbdbb183f627c7c963898044a8d4fdacbf76cc5)